resources/styles.css

```diff
@@ -379,3 +379,20 @@ html, body {
     font-size: small;
     margin-right: 5px;
 }
+
+.context-menu
+{
+    min-width: 100px;
+    display: none;
+    opacity: 1;
+    position: fixed;
+}
+
+.context-menu li.menu-item {
+    min-height: 0px;
+}
+
+.context-menu li.menu-item a {
```

Comment:
Do we really need 3 nested styles? coudln't we simply assign a class to the element we want to style?